src/components/ButtonBar/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { themeSpacing } from '@datapunt/asc-ui';
+
+const Bar = styled.div`
```

Comment:
`ButtonBarStyle`  would be more readable

Reply:
Same here, the result of a call to `styled` is a component, not a list of styled, which the name `ButtonBarStyle` applies